indico/modules/search/client/js/components/results/Category.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {List} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import CategoryPath from './CategoryPath';
+import './Category.module.scss';
+
+export default function Category({title, path, url}) {
```

Comment:
```jsx
export default const Category = ({title, path, url}) => (
  <div styleName="category">
  ...
  </div>
);
```
(once again, personal preference, saves you a `return`)

Reply:
Function naming gets a bit messy when you use arrow functions on the top level as soon as you wrap them in `React.memo()`. Because the one you suggested is actually a syntax error since you cannot have `export default const ...`, just `export default somename;` or `export default <somefunctiondef>;`